src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import { Tab } from '../../react-app-env';
+
+interface TabsProps {
+  onTabSelected:(arg0:Tab)=>void,
+  selectedTabId: string,
+  tabs: Tab[],
+}
+
+export const Tabs: React.FC<TabsProps>
+  = ({ onTabSelected, tabs, selectedTabId }) => {
+    const checkedSelectedTabId
```

Comment:
you can search for tab, not tab id here. This way you will be able to access it for both .id and .content